bam_readcount_helper.py

```diff
@@ -16,13 +16,13 @@ def generate_region_list(hash):
     fh.close()
     return fh.name
 
-def filter_sites_in_hash(region_list, bam_file, ref_fasta, output_dir, insertion_centric):
+def filter_sites_in_hash(region_list, bam_file, ref_fasta, sample, output_dir, insertion_centric):
     bam_readcount_cmd = ['/usr/bin/bam-readcount', '-f', ref_fasta, '-l', region_list, '-w', '0', '-b', '20']
     if insertion_centric:
         bam_readcount_cmd.append('-i')
-        output_file = os.path.join(output_dir, 'bam_readcount_indel.tsv')
+        output_file = os.path.join(output_dir, sample + '_bam_readcount_indel.tsv')
```

Comment:
Will this handle sample names with characters like `/` in them?

Reply:
I am not sure of this. For toil cwl use, the default for sample is either "TUMOR" or "NORMAL".

Reply:
This is fine for that use case, at least!